core/templates/pages/donate-page/donate-page.component.ts

```diff
@@ -49,7 +51,8 @@ export class DonatePageComponent implements OnInit {
   onDonateThroughAmazon() {
     this.siteAnalyticsService.registerGoToDonationSiteEvent('Amazon');
     setTimeout(() => {
-      window.location.href = 'https://smile.amazon.com/ch/81-1740068';
+      this.windowRef.nativeWindow.location.href = (
```

Comment:
Why do we need to change this? Did it not work before?

Reply:
I checked out that using `window` in Angular 2+ service is considered a bad practice, so I've changed it to the `WindowRef`.